src/chocolatey/infrastructure.app/runners/ConsoleApplication.cs

```diff
@@ -85,9 +88,63 @@ public void run(string[] args, ChocolateyConfiguration config, Container contain
                                 }
                             }
                         },
-                        () => command.handle_validation(config),
+                        () =>
+                        {
+                            command.handle_validation(config);
+
+                            var validationResults = new List<ValidationResult>();
+                            var validations = container.GetAllInstances<IValidation>();

```

Comment:
Maybe call this `validationChecks` or something, I'm not super keen on `validations`.